src/messages/formatters.ts

```diff
@@ -1,3 +1,4 @@
+import type { Snowflake } from 'discord-api-types';
```

Comment:
No lone imports. Please specify /common or such instead

Reply:
I'm not sure what you mean. Would you prefer something like `discord-api-types/globals`? Could you elaborate please?

Reply:
`discord-api-types` is not meant to be imported without a subpath, even for types. So, yes, that please